CONTRIBUTORS.md

```diff
@@ -76,7 +76,5 @@ submit the change with your pull request.
 - Brenda Wallace / [br3nda](https://github.com/br3nda)
 - Jim Stallings / [jestallin](https://github.com/jestallin)
 - Alyssa Ransbury / [alran](https://github.com/alran)
-
-## Bots
-
+- Thomas Countz / [thomascountz](https://github.com/thomascountz)
```

Comment:
Oops! I think I was working with an old version of that file. Sorry! 

Is that something I should fix?